.github/workflows/create-release.yml

```diff
@@ -0,0 +1,101 @@
+      env:
+        DOCKERHUB_USERNAME: ${{ secrets.CF_DOCKERHUB_USERNAME }}
+        DOCKERHUB_PASSWORD: ${{ secrets.CF_DOCKERHUB_PASSWORD }}
+      run: |
+        pushd $GITHUB_WORKSPACE/cflinuxfs4
+          echo "${DOCKERHUB_PASSWORD}" | sudo skopeo login --username "${DOCKERHUB_USERNAME}" --password-stdin index.docker.io
+          echo "Making compat stack"
+          make compat version=${{ steps.repo-dispatch.outputs.version }}
+        popd
+
+    - name: BOSH release creation
+      id: bosh-release
+      # TODO change this to main
+      uses: cloudfoundry/buildpacks-github-config/actions/stack/create-release@bosh-release
+      with:
+        version: ${{ steps.repo-dispatch.outputs.version }}
+        access-key-id: ${{ secrets.CF_S3_ACCESS_KEY }}
+        secret-access-key: ${{ secrets.CF_S3_SECRET_ACCESS_KEY }}
+        release-name: cflinuxfs4-compat
+        blob-glob: cflinuxfs4/cflinuxfs4-*.tar.gz
+        blob-name: rootfs
+
+    - name: Git commit blobs.yml
+      id: commit-1
+      uses: paketo-buildpacks/github-config/actions/pull-request/create-commit@main
+      with:
+        message: "Updating blobs for cflinuxfs4-compat bosh release version ${{ needs.create.outputs.version }}"
+        pathspec: "config/blobs.yml"
+        # TODO
+        keyid: ${{ secrets.CF_BOT_GPG_SIGNING_KEY_ID }}
+        key: ${{ secrets.CF_BOT_GPG_SIGNING_KEY }}
+
+    - name: Git commit
+      id: commit-2
+      uses: paketo-buildpacks/github-config/actions/pull-request/create-commit@main
+      with:
+        message: "Final cflinuxfs4-compat bosh release version ${{ needs.create.outputs.version }}, containing cflinuxfs-compat version ${{ needs.create.outputs.version }}"
+        pathspec: ".final_builds releases/**/*-${{ needs.create.outputs.version }}.yml releases/**/index.yml"
+        # TODO
+        keyid: ${{ secrets.CF_BOT_GPG_SIGNING_KEY_ID }}
+        key: ${{ secrets.CF_BOT_GPG_SIGNING_KEY }}
+
+    - name: Git push
+      if: ${{ steps.commit-1.outputs.commit_sha != '' || steps.commit-2.outputs.commit_sha != '' }}
+      uses: paketo-buildpacks/github-config/actions/pull-request/push-branch@main
+      with:
+        branch: "main"
+
+    - name: Create Release
+      uses: paketo-buildpacks/github-config/actions/release/create@main
+      with:
+        repo: ${{ github.repository }}
+        token: ${{ secrets.CF_BOT_GH_TOKEN }}
+        tag_name: v${{ steps.repo-dispatch.outputs.version }}
+        target_commitish: ${{ github.sha }}
+        name: ${{ steps.repo-dispatch.outputs.version }}
+        body: "This release ships cflinuxfs4-compat-release version ${{ steps.repo-dispatch.outputs.version }}, based on [cflinuxfs4](https://github.com/cloudfoundry/cflinuxfs4/releases/tag/${{ steps.repo-dispatch.outputs.version }}) with the addition of python, ruby, and related packages."
+        draft: false
+        assets: releases/cflinuxfs4-compat/cflinuxfs4-compat-${{ steps.repo-dispatch.outputs.version }}.tgz
```

Comment:
doesn't this have to be json? and can you name the release the same as how we do it for cflinxufs4?

Reply:
Oof yea this arg is supposed to be a JSON-encoded list of assets, totally blindly just assumed it could be anything. Do they have to be named the same so that the the compat and normal stacks can be subbed out for each other?

Reply:
I don't think the asset name matters but we should follow other bosh releases convention of naming it <name>-<version>.tgz

Reply:
The name will be `cflinuxfs4-compat-<version>.tgz`

Reply:
fixed in latest commit